documentation/manual/src/main/asciidoc/en-US/modules/redis.asciidoc

```diff
@@ -0,0 +1,1946 @@
+[[ogm-redis]]
```

Comment:
Need your advice on the representation of keys. I gave it a try, but I'm not really happy with the outcome. Don't have a better idea until now.